services/web-app/pages/designing/figma.mdx

```diff
@@ -0,0 +1,176 @@
+<Column colLg={8}>
+
+![figma instructions part 1](/designing/kit-figma-1.png)
+
+</Column>
+</Row>
+
+Then in the Libraries modal that appears, find the team called `[NEW] IBM Design Systems` (C) and
+switch the toggle beside a themed library, for example the `White theme - Carbon Design System` to
+on (D).
+
+<Row>
+<Column colLg={8}>
+
+![figma instructions part 2](/designing/kit-figma-2.png)
+
+</Column>
+</Row>
+
+<br />
+<br />
+
+To preview the four Carbon themes, visit these view-only links.
+
+<Row className="resource-card-group">
+<Column colMd={4} colLg={4} noGutterSm>
+  <ResourceCard
+    onClick={() => fathom.trackGoal('P0OEN9TS', 0)}
+    subTitle="White theme"
+    href="https://www.figma.com/file/Vzz8k68Pqk5HfaTdQOQrGu/White-Theme---Carbon-Design-System"
+    actionIcon="launch">
+
+![Figma icon](/designing/figma.svg)
+
+  </ResourceCard>
+</Column>
+<Column colMd={4} colLg={4} noGutterSm>
+  <ResourceCard
+    onClick={() => fathom.trackGoal('T7D1HJ3L', 0)}
+    subTitle="Gray 10 theme"
+    href="https://www.figma.com/file/i6y80WeotmhmbWxpmcffz7/Gray-10-Theme---Carbon-Design-System"
+    actionIcon="launch">
+
+![Figma icon](/designing/figma.svg)
+
+  </ResourceCard>
+</Column>
+<Column colMd={4} colLg={4} noGutterSm>
+  <ResourceCard
+    onClick={() => fathom.trackGoal('LYFJTPDE', 0)}
+    subTitle="Gray 90 theme"
+    href="https://www.figma.com/file/fCucrHE8tdGDVJX7ODqajd/Gray-90-Theme---Carbon-Design-System"
+    actionIcon="launch">
+
+![Figma icon](/designing/figma.svg)
+
+  </ResourceCard>
+</Column>
+<Column colMd={4} colLg={4} noGutterSm>
+  <ResourceCard
+    onClick={() => fathom.trackGoal('3XH0SIBJ', 0)}
+    subTitle="Gray 100 theme"
+    href="https://www.figma.com/file/TckQzGe3bNxHPLoRhbXFal/Gray-100-Theme---Carbon-Design-System"
+    actionIcon="launch">
+
+![Figma icon](/designing/figma.svg)
+
+  </ResourceCard>
+</Column>
+</Row>
+
+<br />
+
+#### 3. Turn on additional libraries 
+
+Under the same team [New] IBM Design Systems turn on each of the element libraries.
+
+- Color styles - IBM Design Language
+- Text styles - IBM Design Language
+- Icons - IBM Design Language
+- Pictograms - IBM Design Language
+
+You may also turn on the IBM Accessibility Design Kit under IBM Accessibility. Finally, you can
+enable any product or digital design kits your team uses to extend Carbon as needed.
+
+## Start designing
+
+### Components
+
+Included in the library are all 32 of the Carbon components and their variants. To insert a
+component, go to the **Asset** panel and find the component you would like to use. Drag it from the
+asset panel onto the canvas.
+
+View the name of the component in the right sidebar. If the component has variants, you'll see
+fields underneath the component name to configure the properties and values of that component set.
+
+For more help on how to use Figma components, see the
+[Figma docs](https://help.figma.com/hc/en-us/articles/360056440594-Create-and-use-variants#Use_variants).
```

Comment:
Update Component docs link to: https://help.figma.com/hc/en-us/articles/360038662654-Guide-to-components-in-Figma